components/dmesg/component_event.go

```diff
@@ -28,9 +29,11 @@ func ParseEventJSON(data []byte) (*Event, error) {
 const (
 	EventNameDmesgMatched = "dmesg_matched"
 
-	EventKeyDmesgMatchedError = "error"
+	EventKeyDmesgMatchedError   = "error"
+	EventKeyDmesgMatchedLogItem = "log_item"
 )
 
+// TODO: deprecate
```

Comment:
@cardyok We need to deprecate dmesg component, and just use it as a shared poller.

Reply:
make sense to me.
Or we can keep the component (only report state) representing health stage of the poller